content/docs/2.4/scalers/rabbitmq-queue.md

```diff
@@ -37,6 +37,8 @@ triggers:
 - `protocol`: Protocol to be used for communication. Either `auto`, `http`, or `amqp`. It should correspond with the `host` value. Optional, will autodetect based on the `host` URL if possible.
 - `vhostName`: Vhost to use for the connection, overrides any vhost set in the connection string from `host`/`hostFromEnv`.
 - `queueLength`: DEPRECATED! Use `mode: QueueLength` and `value: ##` instead. Target value for queue length passed to the scaler. Example: if one pod can handle 10 messages, set the queue length target to 10. If the actual number of messages in the queue is 30, the scaler scales to 3 pods. Default is 20 unless `publishRate` is specified, in which case `queueLength` is disabled for this trigger.
+- `useRegex`: In case of `http` protocol and `QueueLength` mode, this parameter allows to use regex (in `queueName` parameter) to select queue instead of full name. Optional.
```

Comment:
One last minor thing, we might want to mark the allowed values for `useRegex` parameter, as it might not be clear immediately.

Reply:
I suppose those values are true/false. Should the value be in quotes? Following note suggest that they might be mandatory? https://github.com/kedacore/keda-docs/pull/489/files#diff-4e706b2e67d6b7ef7fb2f4746b593e86326142120d790c6420bb2f785f269dffR51

Reply:
@JorTurFer ^^

Reply:
Maybe I'm wrong, but I understand that all the values in metadata have to be strings to be automatically parsed and accessible  inside the scaler, right? 
You are right, the values are true/false but in order to be automatically parsed, it should be in quotes. It's the same case as `value`, It is a number, but it is converted to string in yaml

Reply:
Yeah, that's true, my comment was more like a hypothetical question from an user's point of view.

Reply:
I can add something like this:
- `useRegex`: In case of `http` protocol and `QueueLength` mode, this parameter allows to use regex (in `queueName` parameter) to select queue instead of full name, the valid values are: `"true"` and `"false"`.  Optional.

WDTY?

Reply:
Yeah, that's looking good!

Reply:
Done ^^